core/services/ocr/contract_tracker.go

```diff
@@ -402,7 +402,7 @@ func (t *OCRContractTracker) LatestBlockHeight(ctx context.Context) (blockheight
 		// care about the block height; we have no way of getting the L1 block
 		// height anyway
 		return 0, nil
-	case "", config.ChainArbitrum, config.ChainCelo, config.ChainOptimismBedrock, config.ChainXDai, config.ChainKroma, config.ChainWeMix, config.ChainZkSync, config.ChainScroll:
+	default:
```

Comment:
The exhaustive style is enforced by the linter. It is meant to make sure future additions are thought about at each call site instead of accidentally falling in to `default`.

Reply:
Done